pkg/webvcs/github.go

```diff
@@ -107,15 +112,42 @@ func (v GithubVCS) ParsePayload(log *zap.SugaredLogger, payload string) (*RunInf
 		return &RunInfo{}, errors.New("cannot parse payload as PR")
 	}
 
-	return &RunInfo{
-		Owner:         prMap.GetRepo().Owner.GetLogin(),
-		Repository:    prMap.GetRepo().GetName(),
-		URL:           prMap.GetRepo().GetHTMLURL(),
-		DefaultBranch: prMap.GetRepo().GetDefaultBranch(),
-		// TODO: this is going to be different on merge
-		SHA:    prMap.PullRequest.Head.GetSHA(),
-		Branch: prMap.PullRequest.Base.GetRef(),
-	}, nil
+	// This should already been set from the rerequest event
```

Comment:
I structured the code like that because I know that we are going to have to parse the issues_comments events next or other type of events,